src/core/agent/services/delegationService.test.ts

```diff
@@ -116,6 +117,28 @@ describe("Delegation sig service of agent", () => {
       { isPending: false }
     );
   });
+  test("should throw error if the identifier.state.di is undefined", async () => {
```

Comment:
I'm being super picky here and only commenting as I've noticed this across our repo. When we add tests can we make sure they are separated with a blank line before and after?

It's a small thing but it makes tests more readable when reviewing a test suite as a whole, otherwise it's easy to get lost